src/test/java/co/omise/live/LiveEventRequestTest.java

```diff
@@ -0,0 +1,78 @@
+package co.omise.live;
+
+import co.omise.Client;
+import co.omise.models.*;
+import co.omise.requests.Request;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+import static org.junit.Assert.*;
+
+public class LiveEventRequestTest extends BaseLiveTest {
+
+    private static final String EVENT_ID = "YOUR_EVENT";
+
+    @Test
+    @Ignore("only hit when test on live.")
+    public void testGetEventSuccess() throws Exception {
+        Client client = getLiveClient();
```

Comment:
How about move `client` to the global variable?

Reply:
Will do 👍

Reply:
I think leave it like this is OK except we want to test on the state sharing part

Reply:
It's way too many calls to get the `Client` though, don't think there's any negative for sharing one client, is there?

Reply:
My concern is the state sharing. Also the those client should be destroyed on test end anyway?

Reply:
I'm not really sure if there's much state to be shared in the client, I get your concern about the `Client` not being fresh when the test happens though. It's not supplied by DI, so I think it gets destroyed as soon as the tests are done anyway (as it's no longer in use)

Reply:
Basically, the `setup()` will call for each test cases. I don't think we gonna have any problem with sharing state.

Reply:
Actually, he's right, it gets called for each test 🤦‍♂️